resources/lang/en/errors.php

```diff
@@ -33,4 +33,7 @@
         'invalid_cursor_exception' => 'Invalid cursor parameter specified.',
         'operation_timeout_exception' => 'Search is currently busier than usual, try again later.',
     ],
+    'user_report' => [
+        'recently_reported' => 'Already reported recently.',
```

Comment:
`You've already reported this recently`.
Type might be nice to have but maybe later 🤷